en/docs/assets/css/blue-palette-alt1.css

```
@@ -9,7 +9,7 @@
     --md-accent-fg-color--transparent: #0339821a;
     --md-accent-bg-color: #fff;
     --md-accent-bg-color--light: #ffffffb3;
-    --md-banner: url(../img/wso2-micro-integrator-hero-image.webp) no-repeat right top;
+    --md-banner: url(../img/home/bi.webp) no-repeat right top;
```

Stylelint (17.3.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)